packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -2958,10 +3000,11 @@ RED.view = (function() {
                                 thisNode.selectAll(".red-ui-flow-node-icon-group").classed("red-ui-flow-node-icon-group-right", false);
                                 thisNode.selectAll(".red-ui-flow-node-label").classed("red-ui-flow-node-label-right", false).attr("text-anchor", "start");
                             }
+                            var gx;
```

Comment:
Please use a meaningful variable name or add a comment so we know what this is used for.